Firebase/DynamicLinks/Public/FDLURLComponents.h

```diff
@@ -525,9 +528,40 @@ FIR_SWIFT_NAME(DynamicLinkComponents)
  * @param link Deep link to be stored in created Dynamic link. This link also called "payload" of
  *     the Dynamic link.
  * @param domain Domain of your App. This value must be equal to your assigned domain from Firebase
- *     Console.
+ *     Console. (e.g. xyz.page.link). Note that the domain scheme is required to be https and is
+ * assumed as such by this API.
  */
-- (instancetype)initWithLink:(NSURL *)link domain:(NSString *)domain;
+- (instancetype)initWithLink:(NSURL *)link
+                      domain:(NSString *)domain
+    DEPRECATED_MSG_ATTRIBUTE(
+        "This method is deprecated. Please use the new method with support for "
+        "domainURIPrefix- initWithLink:domainURIPrefix.");
```

Comment:
Same here - please use the Swift API when referring users to the new API call.

Reply:
Done.